src/nts_ke/client.rs

```diff
@@ -269,7 +270,7 @@ pub fn run_nts_ke_client(
         }
     }
     debug!(logger, "saw the end of the response");
-    stream.shutdown(Shutdown::Both)?;
+    stream.shutdown(Shutdown::Write)?;
```

Comment:
Why not both?

Reply:
Because that doesn't seem to work on NTPSEC.

Try: cfnts client ntp1.glypnod.com -p 123

Reply:
What's happening is the close notify comes in after we shut down the reading half, hence triggering a failure.

Reply:
I see